### PR TITLE
k8s 1.31/go1.23

### PR DESCRIPTION
**Pull request by tjons**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -144,7 +173,9 @@ verify-envoy-protos:
 # Tests
 #----------------------------------------------------------------------------------
 
-GINKGO_VERSION ?= $(shell echo $(shell go list -m github.com/onsi/ginkgo/v2) | cut -d' ' -f2)
+# GINKGO_VERSION ?= $(shell echo $(shell go list -m github.com/onsi/ginkgo/v2) | cut -d' ' -f2)
+# TEMP: Hardcoding version for now, to match go.mod
+GINKGO_VERSION ?= v2.19.0
```
Comment (sam-heilbron) [inline Makefile:178]: Can we undo these temp changes and continue to rely on the go.mod definition to capture the ginkgo version?

**Review by sam-heilbron — COMMENTED** [review]
LGTM! One question about a temp change

File: Makefile (hunk 1 above)
Comment (tjons) [reply Makefile:178]: great idea

**Review by timflannagan — APPROVED** [review]

**Comment by timflannagan** [comment]
Looks like build bot is failing.,

**Review by timflannagan — APPROVED** [review]

**Review by sam-heilbron — APPROVED** [review]

**Pull request merged**